pydantic/json_schema.py

```diff
@@ -148,7 +220,26 @@ def generate_definitions(
         self, inputs: Sequence[tuple[JsonSchemaKeyT, JsonSchemaMode, core_schema.CoreSchema]]
     ) -> tuple[dict[tuple[JsonSchemaKeyT, JsonSchemaMode], DefsRef], dict[DefsRef, JsonSchemaValue]]:
         """
-        Given a list of core_schema, generate all JSON schema definitions, and return the generated definitions.
+        Given a list of core_schema, generates all JSON schema definitions from a list of core schemas, and
+        returns the generated definitions.
```

Comment:
```suggestion
        returns the generated definitions paired with a mapping from the input keys to the definition references
```
This phrasing feels more natural, but weirdly the mapping from input keys to definition references is returned first.

Maybe there's a better way to phrase it for the current order, or maybe it makes sense to return in the opposite order.